golang-github-osbuild-composer.spec

```diff
@@ -68,6 +68,7 @@ export GOFLAGS=-mod=vendor
 %gobuild -o _bin/osbuild-worker %{goipath}/cmd/osbuild-worker
 %gobuild -o _bin/osbuild-tests %{goipath}/cmd/osbuild-tests
 %gobuild -o _bin/osbuild-dnf-json-tests %{goipath}/cmd/osbuild-dnf-json-tests
+%gobuild -o _bin/osbuild-image-tests %{goipath}/cmd/osbuild-image-tests
```

Comment:
Can you add this to `make build` as well, would be useful for running only the tests from a git checkout.